### PR TITLE
uniform style and correct typos

### PR DESCRIPTION
**Pull request by emolinaro**

### CODE REVIEW

File: src/batch/cancel.rst [hunk 1]
```diff
@@ -1,32 +1,40 @@
...
 Canceling Jobs
 ==============
 You can cancel jobs with the ``scancel`` command.
 
-To cancel a single job you can supply the ``jobid`` as the argument.
+To cancel a single job you can supply the ``jobid`` as the argument:
 
 .. code-block:: console
 
    [testuser@fe-ac-02 ~]$ scancel <jobid>
```
Comment (mrlhansen) [inline src/batch/cancel.rst]: If you remove <> everywhere else, you should do the same for `<jobid>` here

File: src/batch/submit.rst [hunk 2]
```diff
@@ -22,11 +26,13 @@ A minimal job script would look something like this:
 
    sort random.txt
 
+|br|
+
 * The first line is a shebang interpreter directive, in this case the interpreter is the *Bash* shell environment.
-* After the shebang any sbatch directives are added. An sbatch directive begins with ``#SBATCH`` followed by the option you want to set. These options can be overriden at execution by supplying them directly to the ``sbatch`` command.
+* After the shebang any sbatch directive is added. An sbatch directive begins with ``#SBATCH`` followed by the option you want to set. These options can be overridden at execution by supplying them directly to the ``sbatch`` command.
```
Comment (mrlhansen) [inline src/batch/submit.rst]: I would change this to "all sbatch directives are added" - there are multiple directives in the example.

File: src/batch/monitor.rst [hunk 3]
```diff
@@ -31,13 +37,15 @@ The ``scontrol`` command can be used to show detailed job information.
 
    [testuser@fe-ac-02 ~]$ scontrol show jobid -dd <jobid>
```
Comment (mrlhansen) [inline src/batch/monitor.rst]: If you remove <> everywhere else, you should do the same for `<jobid>` here

File: src/batch/cancel.rst (hunk 1 above)
Comment (emolinaro) [reply src/batch/cancel.rst]: OK

File: src/batch/submit.rst (hunk 2 above)
Comment (emolinaro) [reply src/batch/submit.rst]: OK

File: src/batch/monitor.rst (hunk 3 above)
Comment (emolinaro) [reply src/batch/monitor.rst]: OK

**Pull request merged**